hack/azure-set-vars.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```

Comment:
Can we add some high-level description of what this script is intended to do? (For posterity)

Reply:
Adding in #131